soc/arm/nordic_nrf/nrf53/soc.c

```diff
@@ -82,6 +83,10 @@ static int nordicsemi_nrf53_init(const struct device *arg)
 	nrf_regulators_dcdcen_vddh_set(NRF_REGULATORS, true);
 #endif
 
+#if defined(CONFIG_SOC_CLOCK_128MHZ_NRF53X_APP)
+	nrf_clock_hfclk_div_set(NRF_CLOCK, NRF_CLOCK_HFCLK_DIV_1);
```

Comment:
`nrfx_clock_divider_set()` should be used instead, and [`clk_init()` in clock_control_nrf.c](https://github.com/nrfconnect/sdk-zephyr/blob/a9423cbce69a5df78140a2cf3ba11aa56d395cfc/drivers/clock_control/clock_control_nrf.c#L623) seems to be the proper place to call this function.